src/main/resources/templates/db_scripts/requestUpdateTrigger.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.${table.tableName}_update() RETURNS TRIGGER AS $$
+
+BEGIN
+  IF OLD.jsonb->>'status' = 'Open - Awaiting pickup' AND
+    NEW.jsonb->>'status' IN ('Closed - Pickup expired', 'Closed - Cancelled') THEN
+    NEW.jsonb = jsonb_set(NEW.jsonb, '{awaitingPickupRequestClosedDate}', to_jsonb(current_timestamp), true);
+  	END IF;
+    RETURN NEW;
+END;
+$$ LANGUAGE 'plpgsql';
+
+
+DROP TRIGGER IF EXISTS ${table.tableName}_update_trigger ON ${myuniversity}_${mymodule}.${table.tableName} CASCADE;
```

Comment:
Why is there a need to cascade the removal of a trigger?

Reply:
Apologies, cascade is redundant here. I copied the example of trigger removal from one of the RMB module's script. Now cascade is removed.